src/handlers/thirdparty.ts

```diff
@@ -44,53 +45,53 @@ export function handleItemUpdated(event: ItemUpdated): void {
     event.params._thirdPartyId,
     event.params._itemId
   )
-  let item = Item.load(itemEntityId)
-  if (item == null) {
+  let itemEntity = Item.load(itemEntityId)
```

Comment:
What do you think of just using `item` here instead of `itemEntity`? The `item` is already an entity